examples/cucumber/build.gradle

```
@@ -10,8 +10,10 @@ dependencies {
     implementation 'org.seleniumhq.selenium:selenium-remote-driver:3.141.59'
     implementation 'org.seleniumhq.selenium:selenium-firefox-driver:3.141.59'
     implementation 'org.seleniumhq.selenium:selenium-chrome-driver:3.141.59'
-    testImplementation 'io.cucumber:cucumber-java:7.11.1'
-    testImplementation 'io.cucumber:cucumber-junit:7.11.1'
     testImplementation 'org.testcontainers:selenium'
-    testImplementation 'org.assertj:assertj-core:3.24.2'
+
+    testImplementation(platform("io.cucumber:cucumber-bom:7.12.0"))
+    testImplementation("io.cucumber:cucumber-java")
+    testImplementation("io.cucumber:cucumber-junit-platform-engine")
+    testImplementation 'io.cucumber:cucumber-junit:7.12.0'
```

if the bom is imported we can remove the version
```suggestion
    testImplementation 'io.cucumber:cucumber-junit'
```